### ISSUE TITLE
using :now option in jwt/unsign requires timestamp rather than DateTime

### ISSUE THREAD

**Issue by rhinoman** [funcool/buddy-sign#40]
The documentation describes a now option that can be used to unsign tokens that may have expired:

```
;; use timestamp in the past
(jwt/unsign token "key" {:now (time/minus (time/now) (time/seconds 5))})
;; => {:user 1}
```
However, I've found this results in the following error:
`org.joda.time.DateTime cannot be cast to java.lang.Number`

As a workaround I've done something like this:
`(jwt/unsign token "key" {:now (buddy.sign.util/to-timestamp (time/minus (time/now) (time/seconds 5)))})`

which works, but it would be more natural to be able to specify a DateTime per the documentation (and would be more consistent with the manner in which the :exp claim is specified during token signing anyway).

**Comment by niwinz** [comment]
Yes, at this moment, the `:now` parameter expects timestamp. But it can be easy to facilitate this and allow DateTime and java.time.Interval instances ;)

**Comment by niwinz** [comment]
Implemented in master and will be released as part of buddy-sign 1.2.0 ;)

**Comment by rhinoman** [comment]
Great.  Thanks!
